coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 90.69
+    "covered_percent": 74.62
   }
```

Comment:
For this and all other files that were auto-generated or that you didn't touch, it's a good rule of thumb to only add the files you worked on instead of just doing `git add .`. I've also been doing this and it makes our commits unmaintainable and a bit hard to read. Let me know if you agree :)